indexer.js

```diff
@@ -34,6 +36,8 @@ router.get('/account/:accountId/activity', findAccountActivity);
 router.get('/account/:accountId/callReceivers', findReceivers);
 router.get('/account/:accountId/likelyTokens', findLikelyTokens);
 router.get('/account/:accountId/likelyNFTs', findLikelyNFTs);
+router.get('/account/:accountId/likelyTokensFromBlock', findLikelyTokensFromBlock);
+router.get('/account/:accountId/likelyNFTsFromBlock', findLikelyNFTsFromBlock);
```

Comment:
```suggestion
router.get('/v2/account/:accountId/likelyTokens', findLikelyTokensFromBlock);
router.get('/v2/account/:accountId/likelyNFTs', findLikelyNFTsFromBlock);
```
Maybe we should expose these as `v2` endpoints instead, the logic between them is essentially the same so would be good to eventually deprecate the old ones. Thoughts?

Reply:
Interesting idea but it's a bit redundant with the `version` param we're returning on the endpoint. I lean on the side of `version` param since I don't want clients to have to update their requests every time the query to the indexer changes where they can instead just rescan with `fromBlockTimestamp` 0 if they detect a version that's different from their cache. I agree that we should deprecate the old ones though. Is there a better way to support this rescan functionality?

Reply:
Ah I see the reason for `version` now, thought it was more related to the endpoint, rather than the internal behavior. Let's leave as is 👍🏽